server/repository/migrations/postgres/2022-10-27T09-15_v103/up.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE user_account ADD COLUMN IF NOT EXISTS "language" varchar NOT NULL DEFAULT 'en';
```

Comment:
`NOT EXISTS` is there because I was testing and ran migration after adding the column.. kept it in to be safe